chart/Chart.yaml

```diff
@@ -14,3 +14,19 @@ dependencies:
   - name: canary-checker
     version: "0.38.158"
     repository: https://flanksource.github.io/charts
+  - name: flanksource-ui
+    version: "1.0.117"
+    repository: https://flanksource.github.io/charts
+    condition: canary-checker-ui.enabled
+    alias: canary-checker-ui
```

Comment:
```suggestion
```

Reply:
This is for existing deployments of canary-checker we have. Otherwise we won't be able to deploy updates to existing canary-checker deployments. \cc @Kaitou786